src/libNode/PoWProcessing.cpp

```diff
@@ -152,7 +153,7 @@ bool Node::StartPoW(const uint64_t& block_num, uint8_t ds_difficulty,
         }
     }
 
-    if (m_state != MICROBLOCK_CONSENSUS)
+    if (m_state != MICROBLOCK_CONSENSUS_PREP || m_state != MICROBLOCK_CONSENSUS)
```

Comment:
This looks familiar. Is it a fix, or an accidental rollback? Just want to make sure we didn't roll back something that was actually fixed.

Reply:
It's a fix. Previously we deleted it for simplicity, however as the MICROBLOCK_CONSENSUS is set when node finished RunConsensusOnMicroBlock, who may also fail, I feel that it's essential to have a PREP state which can act as a buffer to accept finalblock or fallbackblock even though the consensus obj failed to be created. Otherwise the state will always be stuck at  WAITING_DSBLOCK, then the recovery mechanisms such as sync with finalblock or fallbackblock cannot work properly.

Reply:
Always return true?